vita3k/shader/include/shader/usse_utilities.h

```diff
@@ -12,12 +12,10 @@ struct FeatureState;
 namespace shader::usse::utils {
 
 struct SpirvUtilFunctions {
-    spv::Function *pack_f16{ nullptr };
-    spv::Function *unpack_f16{ nullptr };
+    std::map<DataType, spv::Function *> unpack_funcs;
+    std::map<DataType, spv::Function *> pack_funcs;
     spv::Function *pack_fx8{ nullptr };
```

Comment:
fx8 is not similar?

Reply:
I'm not sure about this type yet. I'd like to leave it there for now.